extras/userdoc/md/documentation/an-introduction-to-sli.md

```diff
@@ -74,13 +71,12 @@ to an integer. Using `mark` and `>>`, the content of the userargs array is added
 to a dictionary, which is stored under the name `args`. The second line just
 prints the content of the lamda variable.
 
-SLI user manual
----------------
+## SLI user manual
 
 This manual gives a brief overview of the SLI programming language.
 
-1.  [First Steps](first_steps.md "First Steps")
-2.  [Objects and data types](objects_and_data_types.md "Objects and data types")
-3.  [Programming in SLI](programming_in_sli.md "Programming in SLI")
-4.  [Using files and keyboard input](using_files_and_keyboard_input.md "Using files and keyboard input")
-5.  [Neural simulations](neural_simulations.md "Neural simulations")
+1.  [First Steps](first-steps.md "First Steps")
```

Comment:
Currently, the title is the same as the link text for most links. Therefore, I think that we don't actually need it. We can always re-add it later if we have a use for it. Thus I vote for removing it everywhere.